ajax/libs/animejs/package.json

```diff
@@ -24,7 +24,7 @@
     "target": "git://github.com/juliangarnier/anime.git",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "lib",
         "files": [
           "anime*"
```

Comment:
I think `anime.es.js` should be excluded.

Reply:
Done :)